packages/api/src/command/medical/patient/consolidated-delete.ts

```diff
@@ -64,7 +64,7 @@ async function getResourcesToDelete(
     const resource = r.resource;
     if (!resource) return false;
     // TODO make this dynamic, get a list of resources to exclude
-    return resource.resourceType !== "DocumentReference";
+    return resource.resourceType !== "DocumentReference" && resource.resourceType !== "Patient";
```

Comment:
I think we should exclude the resources not to be deleted before line 52, no?

I understand there's a risk `getConsolidatedPatientData()` returns more than the resources we asked for (I'd say this would be a bug), but then we could check it here "in addition to" doing it there.

One of the reasons is, let's say `getConsolidatedPatientData()` returns only one Patient resource. We won't execute any logic about not having anything to delete (sure, now its just a log, but should we add something between lines 56 and 63 it could be buggy.

But not a huge deal, just trying to make the logic more robust (an even better solution would be to have types limiting the resources that could be asked to delete, but that's another level of refactoring).

Reply:
@leite08 they're already getting filtered at the very top invocation of this - ie the `const resourcesToDelete = resourceTypeForConsolidation.filter(r => r !== "DocumentReference");` line you pointed out

Reply:
I think you didn't understand what I meant, @Goncharo . At line 53 we load resources from the FHIR server. If we know we won't delete DocumentReference and Patient, we might as well not load those in the first place.